osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -321,8 +321,11 @@ protected override void OnDragEnd(DragEndEvent e)
 
         private void selectTab(int index)
         {
-            var channel = ChannelTabControl.Items.Skip(index).FirstOrDefault();
-            if (channel != null && !(channel is ChannelSelectorTabItem.ChannelSelectorTabChannel))
+            var channel = ChannelTabControl.Items
+                                           .Where(tab => !(tab is ChannelSelectorTabItem.ChannelSelectorTabChannel))
+                                           .Skip(index)
+                                           .FirstOrDefault();
```

Comment:
`ElementAtOrDefault` should be enough

Reply:
wow, that exists?!

Reply:
according to the document, it existed from the start of LINQ